components/ui/editor-report/src/main/resources/META-INF/dirigible/editor-report/js/editor.js

```diff
@@ -1389,4 +1389,16 @@ angular.module('page', ['blimpKit', 'platformView', 'platformShortcuts', 'Worksp
 	};
 
 	// End Add from Tables Section ---------------------------------------------------------------------------------
+
+	// Begin Security Section --------------------------------------------------------------------------------------
+
+	$scope.toggleDefaultRoles = function () {
```

Comment:
```suggestion
	$scope.toggleDefaultRoles = () => {
```

Use the same code style as the rest of the application.